client/src/assets/images/components/as-featured-in.tsx

```diff
@@ -1,10 +1,8 @@
-/* eslint-disable max-len */
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const propTypes = { fill: PropTypes.string };
-
-function AsSeenLogo(props) {
+function AsSeenLogo(
```

Comment:
This is not a comment on your work, Parth, but just caught my eye:
- Any reason to not keep the file name and component name in sync?

I would prefer either the filename to be `as-seen-logo` or the component to be `AsFeaturedIn`.

Reply:
I did not rename the component and left it as is. Thought that might be out of the scope of the PR. I'll go ahead and make the changes